src/adapters/springserve.js

```diff
@@ -0,0 +1,105 @@
+var CONSTANTS = require('../constants.json');
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader');
+
+var SpringServeAdapter = function SpringServeAdapter() {
+
+    function buildSpringServeCall(bid){
+
+        var spCall = "http://bidder.springserve.com/display/hbid?";
+
+        //get width and height from bid attribute
+        var size = bid.sizes[0];
+        var width = size[0];
+        var height = size[1];
+
+        spCall += "&w=";
+        spCall += width;
+        spCall += "&h=";
+        spCall += height;
+
+        var params = bid.params;
+        //maps param attributes to request parameters
+        var requestAttrMap = {
+            sp: 'supplyPartnerId',
+            imp_id: 'impId'
+        };
+
+        for (var property in requestAttrMap){
+            if(requestAttrMap.hasOwnProperty && params.hasOwnProperty(requestAttrMap[property])){
+                spCall += '&';
+                spCall += property;
+                spCall += '=';
+                //get property from params and include it in request
+                spCall += params[requestAttrMap[property]];
+            }
+        }
+
+        var domain = window.location.hostname;
+
+        //override domain when testing
+        if(params.hasOwnProperty("test") && params["test"] === true){
+            spCall += "&debug=true";
+            domain = "test.com";
+        }
+
+        spCall += "&domain=";
+        spCall += domain;
+        spCall += "&callback=pbjs.handleSpringServeCB";
+
+        return  spCall
+    }
+
+    function _callBids(params){
+        bids = params.bids || [];
+        for (var i = 0; i < bids.length; i++) {
+            var bid = bids[i];
+            bidmanager.pbCallbackMap[bid.params.impId] = params;
+            adloader.loadScript(buildSpringServeCall(bid))
+        }
+    }
+
+    pbjs.handleSpringServeCB = function(responseObj){
+        if(responseObj && responseObj.seatbid && responseObj.seatbid.length > 0 &&
+            responseObj.seatbid[0].bid[0] !== undefined){
+            //look up the request attributs stored in the bidmanager
+            var responseBid = responseObj.seatbid[0].bid[0];
+            var requestObj = bidmanager.getPlacementIdByCBIdentifer(responseBid.impid);
+            var bid = bidfactory.createBid(1);
+
+            //assign properties from the original request to the bid object
+            for (var i = 0; i < requestObj.bids.length; i++ ){
+                var bidRequest = requestObj.bids[i];
+                if(bidRequest.bidder = "springserve"){
```

Comment:
@JohnBuonomo I noticed this while applying code style tools, but I expect you want this to be an equality check and not assignment. I'll put this fix into the upcoming code style PR.